experimental/web/build_and_test_samples.sh

```diff
@@ -7,17 +7,13 @@
 
 # Builds and tests IREE samples for CI.
 #
-# Accepts no arguments. Configuration must be done via environment variables.
 # The build directory for the emscripten build is taken from the environment
 # variable IREE_EMPSCRIPTEN_BUILD_DIR, defaulting to "build-emscripten".
 # Designed for CI, but can be run manually.
```

Comment:
I did _not_ test these locally. Whoever tries to run these manually might need to troubleshoot a bit.